### PR TITLE
[DB-234] [risk=no] include physical measurements in search results

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. include physical measurements in the search results.

### CODE REVIEW

File: public-ui/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ const routes: Routes = [
         data: {title: 'View Full Results'}
       },
       {
-        path: 'physical-measurements',
+        path: 'physical-measurements/:searchString',
```
Comment (calbach) [inline public-ui/src/app/app-routing.module.ts]: It's a bit more typical to use a query parameter for this kind of thing - you can subscribe to query parameters in the same way you're doing for normal parameters.

e.g. `/physical-measurements?q=asdf`

File: public-ui/src/app/views/pm/pm.component.ts [hunk 2]
```diff
@@ -48,6 +54,10 @@ export class PhysicalMeasurementsComponent implements OnInit, OnDestroy {
     this.loadingStack.push(true);
     this.dbc.getPmGroups().subscribe(results => {
       this.conceptGroups = results;
+      if (this.searchString) {
+        this.conceptGroups = this.conceptGroups.filter(conceptgroup =>
+          conceptgroup.groupName.toLowerCase().includes(this.searchString));
```
Comment (calbach) [inline public-ui/src/app/views/pm/pm.component.ts]: SHould you also be lowercasing the search string?

File: public-ui/src/app/views/quick-search/quick-search.component.ts [hunk 3]
```diff
@@ -189,5 +196,18 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
     localStorage.setItem('searchText', this.prevSearchText);
     this.router.navigateByUrl('/ehr/' + r.domain.toLowerCase());
   }
+  public viewPhysicalMeasurements(searchString: string) {
```
Comment (calbach) [inline public-ui/src/app/views/quick-search/quick-search.component.ts]: nit:newlines?

File: public-ui/src/app/views/quick-search/quick-search.component.ts [hunk 4]
```diff
@@ -189,5 +196,18 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
...
+    this.router.navigateByUrl('/physical-measurements/' + searchString.toLowerCase());
+  }
+  public matchPhysicalMeasurements(searchString: string) {
+      if (searchString) {
```
Comment (calbach) [inline public-ui/src/app/views/quick-search/quick-search.component.ts]: I think this is missing some cases. Probably cleaner would be:

```
if (!this.pmConceptGroups) {
  return 0;
}
if (!searchString) {
 return this.pmConceptGropus.length
}
return filter...
```

Comment (SrushtiGangireddy) [reply public-ui/src/app/views/quick-search/quick-search.component.ts]: Addressed. Thanks.

File: public-ui/src/app/views/quick-search/quick-search.component.ts (hunk 3 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/quick-search/quick-search.component.ts]: Done.

File: public-ui/src/app/views/pm/pm.component.ts (hunk 2 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/pm/pm.component.ts]: Done.

File: public-ui/src/app/app-routing.module.ts (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/app-routing.module.ts]: Changed the param to query param.

File: public-ui/src/app/views/pm/pm.component.ts [hunk 5]
```diff
@@ -35,19 +36,27 @@ export class PhysicalMeasurementsComponent implements OnInit, OnDestroy {
...
 
   ngOnInit() {
+    this.route.queryParams.subscribe(params => {
+      this.searchString = params.searchString || null;
+    });
     this.loadingStack.push(true);
     this.dbc.getPmGroups().subscribe(results => {
       this.conceptGroups = results;
+      if (this.searchString) {
+        this.conceptGroups = this.conceptGroups.filter(conceptgroup =>
```
Comment (calbach) [inline public-ui/src/app/views/pm/pm.component.ts]: camelCase

**Pull request merged**